utils/build.ps1

```diff
@@ -494,6 +568,27 @@ function Fetch-Dependencies {
     Invoke-Program -OutNull msiexec.exe /lvx! $BinaryCache\toolchains\$LogFile /qn /a $BinaryCache\toolchains\WixAttachedContainer\$_ ALLUSERS=0 TARGETDIR=$TARGETDIR
   }
 
+  $NDKURL = "https://dl.google.com/android/repository/android-ndk-r26b-windows.zip"
+  $NDKHash = "A478D43D4A45D0D345CDA6BE50D79642B92FB175868D9DC0DFC86181D80F691E"
+
+  if (-not (Test-Path $BinaryCache\android-ndk-$AndroidNDKVersion-windows.zip)) {
+    Write-Output "android NDK not found. Downloading from google ..."
+    New-Item -ItemType Directory -ErrorAction Ignore $BinaryCache | Out-Null
+    if ($ToBatch) {
```

Comment:
We could extract a "download and verify hash" function that could also take care of $ToBatch (if we still care about that functionality).

Reply:
I think that we do care about the `-ToBatch`, I still often use that to debug things.